hkmc2/shared/src/test/mlscript/nofib/boyer2.mls

```diff
@@ -321,12 +322,41 @@ let rules =
+//│ FAILURE: Unexpected exception
```

Comment:
Whoops. When did that start happening?

Reply:
No idea but seems this it's not related to this PR. I checked on the hkmc2 branch and it also occured